Packages/com.unity.inputsystem/InputSystem/Devices/Sensor.cs

```diff
@@ -248,4 +236,217 @@ protected override void FinishSetup(InputDeviceBuilder builder)
             base.FinishSetup(builder);
         }
     }
+
+    [InputControlLayout(displayName = "Magnetic Field")]
+    public class MagneticFieldSensor : Sensor
+    {
+        /// <summary>
+        /// TODO
+        /// </summary>
+        /// <remarks>
+        /// Values are in micro-Tesla (uT) and measure the ambient magnetic field in the X, Y and Z axis.
```

Comment:
I'm surprised to see specific physical units for all these added sensors. Can we actually assume to get specific physical measurements from all/most implementations (instead of axes of arbitrary scale)?

Reply:
Don't know TBH. So far, these are all taken directly from Android which is the only platform where we currently support all these sensors. When extending sensor support, this needs to be looked at.

Reply:
Well, guess since it's only comments in the code, doesn't really matter anyways. Was just curious.